bullet_train/app/views/account/memberships/_form.html.erb

```diff
@@ -1,42 +1,44 @@
 <%= form_with(model: [:account, (@team unless membership.persisted?), membership], class: 'form', local: true) do |form| %>
   <%= render 'account/shared/forms/errors', form: form %>
 
-  <% if membership.unclaimed? || membership.tombstone? %>
-    <div class="grid grid-cols-1 gap-y gap-x sm:grid-cols-6">
-      <div class="sm:col-span-3">
-        <%= render 'shared/fields/text_field', form: form, method: :user_first_name %>
-      </div>
 
-      <div class="sm:col-span-3">
-        <%= render 'shared/fields/text_field', form: form, method: :user_last_name %>
+  <% with_field_settings form: form do %>
```

Comment:
This block looks way messier than it really is. I just wrapped and entire section in:

```ruby
  <% with_field_settings form: form do %>
  <% end %>
```

Turning off whitespace in the diff will make this much easier to review.